apps/web/next.config.js

```diff
@@ -96,6 +96,26 @@ const nextConfig = {
   },
 
   productionBrowserSourceMaps: true, // Let's see if this helps with Sentry... We are OSS anyways so doesn't matter too much if source code is "leaked" to client
+  headers: async () => [
+    {
+      source: "/",
+      headers: [
+        {
+          key: "Access-Control-Allow-Origin",
+          value: "*", // Allow any origin 
```

Comment:
I think there are some situations where it wants an actual hostname and does not like wildcard

Reply:
specifically for Authorization headers, but we added it to Access-Control-Allow-Headers

https://github.com/AirenSoft/OvenMediaEngine/discussions/1616

However, our API keys are in x-apikey so browsers should not get in the way of us. 

I tested this so lets merge it. If we have a user saying it doesn't work we can expand on this and do it in code vs just the JSON config .